packages/app/src/renderer/models/settings/settings.test.ts

```diff
@@ -473,7 +472,7 @@ test("updates language setting", async () => {
   `)
 })
 
-test("sendDiagnosticData effect no generate any side effects if serial number is undefined", async () => {
+test.skip("sendDiagnosticData effect no generate any side effects if serial number is undefined", async () => {
```

Comment:
What about these skips? maybe we should add a task to check this out?

Reply:
https://appnroll.atlassian.net/browse/CP-884